packages/kotti-ui/source/kotti-field-select/KtFieldSingleSelect.vue

```diff
@@ -25,6 +25,16 @@
 					:label="option.label"
 					:value="option.value"
 				/>
+				<div v-if="actions" class="kt-field-select-actions">
```

Comment:
condition should be explicit to only show when array length is more than 0 so: `actions.length`, 
i see that ur default prop value is null, which justifies ur choice of condition but i vote for `[]` and checking on length b/c you expect an array to fallback to `[]` and then it's clear to the reader w/o going through further code, that this is an array

as far as i'm concerned, that's not BEM, so maybe:`kt-field-select__actions`